lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -44,14 +44,19 @@ function subscribeEvents() {
   // Signup button on pitch page
   const isPitchPage = $('.pitch').length > 0;
   if (isPitchPage) {
+    $('#link--campaign-signup-login').on('click', () => {
```

Comment:
What's the difference in this event rather than the existing `Modal:Open` analytics handler?